__tests__/text-classification.spec.js

```diff
@@ -32,12 +32,22 @@ test('text classification simple', async () => {
   await page.click('text=Loggers')
   await page.click('text=config.yaml')
 
-  await page.getByRole('button', { name: 'Code' }).click()
   const downloadPromise = await page
     .waitForEvent('download', { timeout: 2000 })
     .catch(() => {
+      page.getByRole('button', { name: 'Code' }).click()
```

Comment:
I do not understand why you do not call 
```
page.getByRole('button', { name: 'Code' }).click()
page.getByRole('button', { name: 'Download Zip' }).click()
```
before download promise:
```
  const downloadPromise = await page
    .waitForEvent('download', { timeout: 2000 })
```
?

It does not seem correct to reclick 3 times in `catch`